docs/content/docs/connectors/table/upsert-kafka.md

```diff
@@ -49,6 +49,9 @@ Dependencies
 
 {{< sql_download_table "upsert-kafka" >}}
 
+The Upsert Kafka connector is not currently part of the binary distribution.
```

Comment:
remove _currently_, as it implies that at some point in the future it will be part of the distribution. Same for other changes to connectors pages